File: src/main/java/com/saybetter/domain/symbol/exception/SymbolException.java
package com.saybetter.domain.symbol.exception;

import com.saybetter.global.common.code.BaseErrorCode;
import com.saybetter.global.common.exception.GeneralException;

public class SymbolException extends GeneralException {

	public SymbolException(BaseErrorCode code) {
		super(code);
	}

}

File: core/core-api/src/main/java/io/say/better/domain/symbol/exception/SymbolException.kt
package io.say.better.domain.symbol.exception;

import io.say.better.global.common.code.BaseErrorCode;
import io.say.better.global.common.exception.GeneralException;

public class SymbolException extends GeneralException {

	public SymbolException(BaseErrorCode code) {
		super(code);
	}

}
